GOVERNANCE.md

```diff
@@ -0,0 +1,95 @@
+# Roles, People, and Processes for Maintaining iD
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project.
+
+The maintainer role includes:
+
+- Having the final say in decisions for the project.
+- Creating releases.
+- Updating dependencies.
+- Assign roles.
+
+and all the following roles.
```

Comment:
`and all the following roles.` sounds a bit unclear, though understandably

Maybe "and all what is included in the following roles."

Or "and all responsibilities of the following roles."

? Or something like that?

Reply:
Thanks, I updated the two lines.